python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,55 @@
+from ray._private.structured_logging.formatters import TextFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+from typing import Union
+
+# A dictionary to map encoding types to their corresponding logging configurations.
+LOG_MODE_DICT = {
+    "TEXT": {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "text": {
+                "()": TextFormatter,
+            },
+        },
+        "filters": {
+            "core_context": {
+                "()": CoreContextFilter,
+            },
+        },
+        "handlers": {
+            "console": {
+                "level": "INFO",
+                "class": "logging.StreamHandler",
+                "formatter": "text",
+                "filters": ["core_context"],
+            },
+        },
+        "root": {
+            "level": "INFO",
+            "handlers": ["console"],
+        },
+    },
+}
+
+
+class LoggingConfig:
+    def __init__(self, log_config: Union[dict, str]):
```

Comment:
As we discussed, let's do

```
class LoggingConfig:
    dict_config: Union[dict, str] = "TEXT"
    # Only effective if dict_config is not a dict
    log_level: int = INFO
```

Reply:
Updated https://github.com/ray-project/ray/pull/45344/commits/a38ec102e0d2b2a7666b983f6ed0c5cd73fc008e